### PR TITLE
Fixed Staring Page

### PR DESCRIPTION
**Pull request by EthanAuyeung**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Fixed so that when a user navigates to the site the first page that they see is the login page and not just a blank screen with the word login on it.

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # "Optional - copy links to any tutorials or documentation that was useful to you when working on this PR"

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # "This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation."
🛍 CC: @EthanAuyeung

### CODE REVIEW

File: src/app/cart/page.tsx [hunk 1]
```diff
@@ -73,14 +74,16 @@ export default function OrderPage() {
...
-            Check Out
-          </CheckoutButton>
+          <OrderSumSectionSpacing>
+            <OrderSummary cart={cart} numberOfItems={numberOfItems} />
+            <CheckoutButton
+              // change this function so that the flow makes sense and that there is items within the cart
+              onClick={() => checkDelivery()}
+              disabled={numberOfItems === 0}
+            >
+              Check Out
```
Comment (BuyankhuuTsCAl) [inline src/app/cart/page.tsx:84]: can you change this to make sure that the button has styling on the text?

Comment (BuyankhuuTsCAl) [reply src/app/cart/page.tsx:84]: also the wording changed to place order?

File: src/app/cart/styles.tsx [hunk 2]
```diff
@@ -65,8 +65,9 @@ export const Backtext = styled.p`
 export const TrashIcon = styled(Trash2)`
   width: 30px;
   height: 30px;
-  color: black;
+  color: navy;
```
Comment (BuyankhuuTsCAl) [inline src/app/cart/styles.tsx:68]: use the color json!

File: src/app/orderConfirmationDelivery/page.tsx [hunk 3]
```diff
@@ -62,6 +63,9 @@ export default function OrderConfirmationDelivery() {
       <NavBar />
       <CenterDiv>
         <PageDiv>
+          <BackButtonDiv>
+            <BackButton destination="./storefront" />
+          </BackButtonDiv>
           <BottomColumnDiv>
             <LeftColumnDiv>
               <BackButton destination="./storefront" />
```
Comment (BuyankhuuTsCAl) [inline src/app/orderConfirmationDelivery/page.tsx:71]: is there two backbuttons?

File: src/app/orderConfirmationDelivery/page.tsx [hunk 4]
```diff
@@ -81,8 +85,25 @@ export default function OrderConfirmationDelivery() {
                         }}
                       />
                       <LabelBox>
-                        <Body1Bold>{cartItem.name}</Body1Bold>
-                        <Body2Light>Category: {cartItem.category}</Body2Light>
+                        <div>
+                          <span
+                            style={{
+                              display: 'inline-block',
+                              fontWeight: 'bold',
```
Comment (BuyankhuuTsCAl) [inline src/app/orderConfirmationDelivery/page.tsx:92]: don't do inline styling! I feel like we should keep in consistent

File: src/app/orderConfirmationDelivery/page.tsx [hunk 5]
```diff
@@ -81,8 +85,25 @@ export default function OrderConfirmationDelivery() {
...
+                              marginRight: '4px',
+                            }}
+                          >
+                            Quantity:
+                          </span>
+                          <span
+                            style={{
```
Comment (BuyankhuuTsCAl) [inline src/app/orderConfirmationDelivery/page.tsx:99]: same idea

File: src/app/orderConfirmationPickUp/page.tsx [hunk 6]
```diff
@@ -116,10 +115,30 @@ export default function OrderConfirmationPickUp() {
                       <LabelBox1>
                         <Body1Bold>{cartItem.name}</Body1Bold>
                         <br />
-                        <Body2Light>Category: {cartItem.category}</Body2Light>
+                        <Body2Light style={{ marginTop: '-10px' }}>
```
Comment (BuyankhuuTsCAl) [inline src/app/orderConfirmationPickUp/page.tsx:118]: ideally avoid inline

File: src/app/orderConfirmationPickUp/page.tsx [hunk 7]
```diff
@@ -116,10 +115,30 @@ export default function OrderConfirmationPickUp() {
...
+                          Category: {cartItem.category}
+                        </Body2Light>
                       </LabelBox1>
                       <LabelBox>
-                        <Body1Bold>Quantity: {cartItem.quantity}</Body1Bold>
+                        <div>
+                          <span
+                            style={{
+                              display: 'inline-block',
```
Comment (BuyankhuuTsCAl) [inline src/app/orderConfirmationPickUp/page.tsx:126]: same here
